pandas/core/window.py

```diff
@@ -602,6 +604,8 @@ def validate(self):
         if isinstance(window, (list, tuple, np.ndarray)):
             pass
         elif is_integer(window):
+            if window == 0:
```

Comment:
Instead, you can combine this check with the one below (i.e `window <= 0`) and then raise an appropriate error message